changelogs/fragments/60508-route53-improve-octal-characters-handling.yml

```
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

If you want this to be backported, it's better to also label it a bugfix in the changelog fragment:
```suggestion
bugfixes:
```